client/llb/exec.go

```diff
@@ -92,12 +93,12 @@ func (e *ExecOp) Validate() error {
 	return nil
 }
 
-func (e *ExecOp) Marshal() ([]byte, error) {
+func (e *ExecOp) Marshal() ([]byte, *pb.OpMetadata, error) {
```

Comment:
I'd just `(*llb.Definition, error)`. This can be follow-up

Reply:
This function just returns `[]byte` representation of single Op while `llb.Definition` contains `[][]byte` representation of multiple Ops.
Should we modify definition of `llb.Definition`?

Reply:
ah, sorry. I mixed this up with https://github.com/moby/buildkit/pull/114/files#diff-23cdf6d75aab7a60db2eaa3d2b59710eR51 . No change required.